lib/MFA.php

```diff
@@ -112,11 +112,11 @@ public function challengeFactor(
      * @param string $code - The verification code sent to and provided by the end user.
     */
 
-    public function verifyFactor(
+    public function verifyChallenge(
         $authenticationChallengeId,
         $code
     ) {
-        $verifyPath = "auth/factors/verify";
+        $verifyPath = "auth/challenges/{$authenticationChallengeId}/verify";
```

Comment:
Could we remove the `authentication_challenge_id` entry from the `params` below?